Source/WebKit/UIProcess/Automation/protocol/BidiBrowsingContext.json

```diff
@@ -0,0 +1,61 @@
+{
+    "domain": "BrowsingContext",
+    "description": "The browsingContext module contains commands and events relating to navigables.",
+    "condition": "ENABLE(WEBDRIVER_BIDI)",
+    "types": [
+        {
+            "id": "BrowsingContext",
+            "type": "string",
+            "description": "A unique identifier for a navigable."
+        },
+        {
+            "id": "Navigation",
+            "type": "string",
+            "description": "Unique string identifying an ongoing navigation."
+        },
+        {
+          "id": "ReadinessState",
+          "type": "string",
+          "description": "Represents the stage of document loading at which a navigation command will return.",
+          "enum": ["none", "interactive", "complete"]
+        }
+    ],
+    "commands": [
+        {
+            "name": "navigate",
+            "description": "Navigates a navigable to the given URL.",
+            "parameters": [
+                {
+                    "name": "context",
+                    "$ref": "BrowsingContext",
+                    "description": "The identifier of the browsing context to navigate."
+                },
+                {
+                    "name": "url",
+                    "type": "string",
+                    "description": "The URL to navigate to."
+                },
+                {
+                    "name": "wait",
+                    "$ref": "BrowsingContext.ReadinessState",
+                    "optional": true,
+                    "description": "The readiness state at which the command will return (default is 'none')."
+                }
+            ],
+            "returns": [
+                {
+                    "name": "navigation",
+                    "$ref": "BrowsingContext.Navigation",
+                    "optional": true,
```

Comment:
it's kinda odd to have optional things come before non-optional stuff, so id recommend flipping the order of this with `url`